docs/internals/services.rst

```diff
@@ -1,6 +1,112 @@
-.. include:: /_shared/stub-topic.txt
+Understanding EF Services
+=========================
 
-|stub-icon| Understanding EF Services
-=====================================
+.. contents:: `In this article`
 
-.. include:: /_shared/stub-notice.txt
+Entity Framework executes as a collection of services working together. A
+service is a reusable component. In C#, a services is typically an
```

Comment:
Not sure about saying this is typical is in the scope of C#. Maybe just remove that?